apps/dashboard/app/views/system_status/index.turbo_stream.erb

```diff
@@ -9,12 +9,13 @@
 
           <% components_status(job_adapter).each do |current_status| %>
             <div class="container py-2">
-              <div><%= current_status[:message] %></div>
+              <span><%= current_status[:message] %></span>
 
               <% if valid_percent?(current_status[:percent]) %>
                 <div class="progress">
-                  <div class="progress-bar" role="progressbar" style="width: <%= current_status[:percent] %>"><%= current_status[:percent] %> in use</div>
+                  <div class="progress-bar" role="progressbar" style="width: <%= current_status[:percent] %>"></div>
```

Comment:
Thanks. I was looking into this too. Can we also change this inline style to `w-<%= current_status[:percent].to_i %>` class? we have `w-[1-100]` css classes.